File: Bedu/DesarrolloMobileJava/Examen/Examen/app/src/main/java/xyz/eliothmonroy/examen/utils/RetrofitUtils.java
package xyz.eliothmonroy.examen.utils;

import retrofit2.Retrofit;
import retrofit2.converter.gson.GsonConverterFactory;

public class RetrofitUtils {
    public static Retrofit getRetrofitInstance(){
        return new Retrofit.Builder().baseUrl(CONSTANTS.BASE_URL_API)
                .addConverterFactory(GsonConverterFactory.create()).build();
    }
}

File: Bedu/KotlinFundamentals/Sesion8/Examen/app/src/main/java/xyz/eliothmonroy/examen/utils/RetrofitUtils.kt
package xyz.eliothmonroy.examen.utils

import retrofit2.Retrofit
import retrofit2.converter.gson.GsonConverterFactory

object RetrofitUtils {

    private val retrofit: Retrofit = Retrofit.Builder().baseUrl(CONSTANTS.BASE_URL).addConverterFactory(
        GsonConverterFactory.create()).build()

    fun getRetofitInstance():Retrofit{
        return retrofit
    }

}